src/main/cpp/t10_unique.cpp

```diff
@@ -21,9 +21,21 @@
 
 #include "t10_unique.h"
 #include <iostream>
-
+#include <vector>
 using namespace std;
 
 int t10_unique() {
-
+int a[100], n;
+cin >> n;
+for (int i=0; i<n; i++){
+cin >> a[i];
+}
+int z;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'z' can be reduced.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4719943)